### DISCUSSION TITLE
Virtual Audio Device Driver on Windows

### DISCUSSION THREAD

**Discussion by mkckr0** [mkckr0/audio-share#59] [General]
**"Steam Streaming Speakers" Valve Corp.**
When user uses Steam Link to connect to PC, Steam will install the driver automatically.
People who doesn't want to install Steam can download the zip and install manually.
It's copied from `C:\Windows\System32\DriverStore\FileRepository\steamstreamingspeakers.inf_amd64_78513697685d0fe1`
[SteamStreamingSpeakers.zip](https://github.com/mkckr0/audio-share/files/14686557/SteamStreamingSpeakers.zip)

**Comment by Midi-In** [comment]
There are a few different programs from VB-Audio that are used exactly for this purpose - audio endpoint virtualisation and advanced mixing capabilities. The simplest one is https://vb-audio.com/Cable/

**Comment by Midi-In** [comment]
There's also this, although it's paid https://vac.muzychenko.net/en/
